pkgs/by-name/au/autobase/package.nix

```diff
@@ -27,7 +33,60 @@ buildNpmPackage (finalAttrs: {
     cp ${./package-lock.json} ./package-lock.json
   '';
 
-  passthru.updateScript = nix-update-script { };
+  passthru = {
+    updateScriptSrc = gitUpdater {
+      rev-prefix = "v";
+    };
+    updateScriptVendor = writeShellApplication {
+      name = "update-autobase-lockfile-npmDepsHash";
+      runtimeInputs = [
+        nix
+        nodejs
+        prefetch-npm-deps
+        sd
+      ];
+      text = ''
```

Comment:
Those long scripts should probably be in an extra update script

Reply:
You mean a file? Sure, can do.